adafruit_epd/ssd1680.py

```diff
@@ -24,7 +24,7 @@
 except ImportError:
     pass
 
-__version__ = "0.0.0+auto.0"
+__version__ = "2.13.0"
```

Comment:
dont do this

Reply:
The version will be filled in by the bundling tools. Leave it as `0.0.0+auto.0`.